include/inicpp.h

```diff
@@ -577,19 +586,29 @@ namespace ini
             esc_ = esc;
         }
 
+        /** Sets whether or not to parse multi-line field values.
+          * Default is false.
+          * @param enable enable or disable? */
+        void setMultiLineValues(bool enable)
+        {
+            multiLineValues_ = enable;
+        }
+
         /** Tries to decode a ini file from the given input stream.
           * @param is input stream from which data should be read. */
         void decode(std::istream &is)
         {
             this->clear();
             int lineNo = 0;
             IniSectionBase<Comparator> *currentSection = nullptr;
+            std::string multiline_value_field_name = "";
```

Comment:
Use camelCase for the variable name to stick to naming convention.

Reply:
I've adjusted these. Let me know if I've missed anything.